apps/files/src/services/DropService.ts

```diff
@@ -67,6 +70,14 @@ export const handleDrop = async (data: DataTransfer): Promise<Upload[]> => {
 const handleFileUpload = async (file: File, path: string = '') => {
 	const uploader = getUploader()
 
+	const toCheck = file.name.split('')
+	toCheck.forEach(char => {
+		if (forbiddenCharacters.indexOf(char) !== -1) {
+			showError(t('files', t('files', '"{char}" is not allowed inside a file name.', { char })))
+			throw '#.. is not allowed inside a file name.'
```

Comment:
Throwing a string is nothing we should do, instead a proper `Error` should be used.

```suggestion
			throw new Error(`${char} is not allowed inside a file name.`).
```